pkgs/tools/virtualization/kubevirt/default.nix

```diff
@@ -0,0 +1,44 @@
+{ buildGoModule
+, fetchFromGitHub
+, lib
+, testers
+, kubevirt
+}:
+
+buildGoModule rec {
+  pname = "kubevirt";
+  version = "0.52.0";
+  strictDeps = true;
```

Comment:
```suggestion
```
Thats the default already https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/go-modules/generic/default.nix#L263